CHANGELOG

```diff
@@ -1,3 +1,6 @@
+# 3.0.17
```

Comment:
Merged daniels PR @ v3.0.17 so this will need to be bumped to v3.0.18

Reply:
Yes, would rebase to last merge